test-network-function/platform/suite.go

```diff
@@ -243,16 +243,16 @@ func testBootParamsHelper(context *interactive.Context, podName, podNamespace st
 
 func testSysctlConfigs(env *config.TestEnvironment) {
 	ginkgo.It("platform-sysctl-config", func() {
-		context := common.GetContext()
-		for _, cut := range env.ContainersUnderTest {
-			podName := cut.Oc.GetPodName()
-			podNameSpace := cut.Oc.GetPodNamespace()
-			testSysctlConfigsHelper(context, podName, podNameSpace)
+		for _, podUnderTest := range env.PodsUnderTest {
+			podName := podUnderTest.Name
+			podNameSpace := podUnderTest.Namespace
+			testSysctlConfigsHelper(podName, podNameSpace)
 		}
 	})
 }
-func testSysctlConfigsHelper(context *interactive.Context, podName, podNamespace string) {
+func testSysctlConfigsHelper(podName, podNamespace string) {
```

Comment:
Took a closer look at this. It is really a node level test, similar to some others you converted. I am gonna add this to the node list jira as a reference.